ragna/deploy/_ui/central_view.py

```diff
@@ -225,11 +276,17 @@ def on_click_chat_info_wrapper(self, event):
                     # The CSS rule below relies on a variable value, so we can't move it into modifers
                     stylesheets=[
                         ui.css(
-                            ":host(.chat_info_markdown) .pills_list",
+                            ":host(.chat_info_markdown) .details",
                             {
                                 "grid-template": f"repeat({grid_height}, 1fr) / repeat(3, 1fr)",
                             },
-                        )
+                        ),
+                        ui.css(
+                            ":host(.chat_info_markdown) .details_block",
+                            {
+                                "display": "block",
+                            },
+                        ),
```

Comment:
Let's shed a few commas so the auto formatter unexplodes this

```suggestion
                        ui.css(
                            ":host(.chat_info_markdown) .details_block"
                            {
                                "display": "block"
                            }
                        ),
```

Reply:
Erf, I shouldn't have committed this directly from Github's interface, I didn't see there was a missing comma at the end of line 290. I'm fixing this now.